src/compute-pr-actions.ts

```diff
@@ -339,7 +339,7 @@ export function process(prInfo: BotResult,
     // Has it: got no DT tests but is approved by DT modules and basically blocked by the DT maintainers - and it has been over 3 days?
     // Send a message reminding them that they can un-block themselves by adding tests.
     if (!info.hasTests && !info.hasMultiplePackages && info.approvedBy.includes("owner") && !info.editsInfra
-        && info.approverKind === "maintainer" && (info.staleness?.days ?? 0) > 3) {
+        && info.approverKind === "maintainer" && dayjs(info.now).diff(info.lastPushDate, "days") > 3) {
```

Comment:
We don't apply the unreviewed timeline to these PRs anymore (ones in the maintainer queue with reviews). This replacement is the same as if we did.